src/sentry/api/helpers/group_index/update.py

```diff
@@ -195,15 +205,13 @@ def update_groups(
     serializer = validate_request(request, projects, data)
 
     if serializer is None:
-        return
```

Comment:
this is actually the last of a particular class of error that I've been avoiding because it's very non-obvious what the right thing to do here is.  I'm pretty sure this crashes in a weird way if this line were ever reached?